.circleci/config.yml

```diff
@@ -88,6 +88,8 @@ jobs:
           environment:
             MLFLOW_PIPELINES_PROFILE: local
           command: |
+            # Patch for custom_metrics.py to make it return a scalar value
+            cat ../../../dev/custom_metrics_patch.py > steps/custom_metrics.py
```

Comment:
A temporary fix until https://github.com/harupy/mlp-regression-example/pull/1 is merged.